preview-src/cache.ts

```diff
@@ -30,6 +30,7 @@ export interface PullRequest {
 	status: ReposGetCombinedStatusForRefResponse;
 	mergeable: boolean;
 	defaultMergeMethod: string;
+	availableMergeMethods: string[];
```

Comment:
Should I create separate union type for available merge methods instead?
If yes - which file is the best to add this type?

Reply:
I think that would be good to have - `src/github/interface.ts` is a good place for that